src/api/python/cvc5.pxi

```diff
@@ -207,32 +208,57 @@ cdef class DatatypeConstructor:
         return ds
 
     def getName(self):
+        """
+            :return: the name of the constructor.
+        """
         return self.cdc.getName().decode()
 
     def getConstructorTerm(self):
+        """
+            :return: the constructor operator as a term.
+        """
         cdef Term term = Term(self.solver)
         term.cterm = self.cdc.getConstructorTerm()
         return term
 
     def getSpecializedConstructorTerm(self, Sort retSort):
+        """
+            Specialized method for parametric datatypes (see :cpp:func:`DatatypeConstructor::getSpecializedConstructorTerm() <cvc5::api::Datatype::getSpecializedConstructorTerm>`).
```

Comment:
This link does not work, apparently. Also, it says `DatatypeConstructor` but links to `Datatype`. Should it go to `getConstructorTerm`?